release-notes-merge-action.py

```diff
@@ -41,22 +40,23 @@
 
     pre_release_bool = False
 
+    addon_version = "music_assistant"
+
     if "b" in args.release_tag:
         pre_release_bool = True
```

Comment:
cant we pass this from the release script as argument ?

Reply:
I guess I always did it this way, and didn't think too much about it back when I first wrote this stuff, but it makes far more sense to pass this in, as it must be available